.github/workflows/template_gitops.yml

```diff
@@ -9,6 +9,10 @@ on:
       dockerbuildtarget:
         required: false
         type: string
+      dockerenabled:
```

Comment:
Could you add this option to the readme?

Reply:
Of course! 06b16635172adbff32e96a81fb42c9e6539374b6